ftw/activity/tests/test_uid_check.py

```diff
@@ -0,0 +1,34 @@
+from ftw.activity.subscribers import is_supported
+from ftw.activity.testing import FUNCTIONAL_TESTING
+from ftw.builder import Builder
+from ftw.builder import create
+from plone.app.testing import TEST_USER_ID
+from plone.app.testing import setRoles
+from unittest2 import TestCase
+
+
+class TestUIDCheck(TestCase):
+    # is_supported checks if the object of the event can be used
+    # to create an activity event.
+    # This prevents errors in Plone when moving objects whose actions
+    # cannot be indexed by ftw.activity.
+
+    layer = FUNCTIONAL_TESTING
+
+    def setUp(self):
+        setRoles(self.layer['portal'], TEST_USER_ID, ['Manager'])
+        self.portal = self.layer['portal']
+
+    def test_plone_site_is_not_supported(self):
+        is_supported(self.portal)
```

Comment:
I think you forgot the assertions in all the tests 😜 
Use `assertTrue()` and `assertFalse()`